<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="false">
 <class>EEGsignalsClass</class>
 <widget class="QMainWindow" name="EEGsignalsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EEGsignals</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>setAlphaChannel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>setThetaChannel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="CustomPlot" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>451</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="EEGAlphaSignalPlot" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>451</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EEGsignals.qrc"/>
 </resources>
 <connections/>
</ui>
